docs/content/preview/yugabyte-cloud/cloud-admin/manage-access.md

```diff
@@ -33,12 +33,24 @@ Admin users have full access to all features. There must always be at least one
 
 Developer users have access to all features, with the exception of administrative tasks, including the following:
 
-- invite users.
-- delete or change the role of other users.
-- create a billing profile.
+- invite users
+- delete or change the role of other users
+- change login methods
+- create a billing profile
+- view account activity
 
 To access a cluster database, you need to ask an Admin for the username and password of a [database user created on your behalf](../../cloud-secure-clusters/add-users/).
 
+## Manage login methods
+
+Users can log in to YugabyteDB managed using either an email-based account or a social login. The available social logins include Google, GitHub, and LinkedIn. All three are enabled by default. Only an Admin user can modify the login methods.
```

Comment:
YugabyteDB managed  >  YugabyteDB Managed

Reply:
We need to mention that if the admin has revoked a particular login method (say Google), users can use the "Forgot password" option to configure email based login (if not configured already)